experimental/traffic-portal/src/app/core/core.module.ts

```diff
@@ -107,14 +112,17 @@ export const ROUTES: Routes = [
 		DivisionDetailComponent,
 		RegionsTableComponent,
 		RegionDetailComponent,
-		CacheGroupDetailsComponent
+		CacheGroupDetailsComponent,
+		StatusesTableComponent,
+		StatusDetailsComponent
 	],
 	exports: [],
 	imports: [
 		SharedModule,
 		AppUIModule,
 		CommonModule,
-		RouterModule.forChild(ROUTES)
+		RouterModule.forChild(ROUTES),
+		ReactiveFormsModule
```

Comment:
The `AppUIModule` imports and exports `ReactiveFormsModule`, so you shouldn't need to add this.